projects/kit/components/rating/rating.style.less

```diff
@@ -0,0 +1,45 @@
+@import 'taiga-ui-local';
+
+:host {
+    position: relative;
+    display: inline-flex;
+}
+
+.t-rating {
+    display: inline-flex;
+    flex-direction: row-reverse;
+
+    &__input {
+        position: absolute;
+        top: 0;
+        left: 0;
+        bottom: 0;
+        cursor: pointer;
+        width: 100%;
+        appearance: none;
+        background: transparent;
```

Comment:
is it good for a11y ?
This input will not be detected by screen readers (otherwise use `.sr-only()` mixin)

I see that  you added
```ts
    [attr.aria-label]="value + ' Stars'"
```
Should we hide all screen-reader's features with pronunciation of `<input type="range">` ?

Reply:
**btw**
I am not insist on this.
Just wants to discuss it.

Reply:
`.sr-only()` If I add this mixin then focus stops working, I added a comment in styles